core/src/main/resources/jenkins/security/apitoken/LegacyApiTokenAdministrativeMonitor/manage.properties

```diff
@@ -4,24 +4,24 @@ contextMessage=The following users have a legacy API token. \
   Additionally, in previous versions of Jenkins, API tokens were automatically created for every new user. \
   Often, users did not use the token, resulting in a larger attack surface than necessary. \
   Automatic token generation must now be enabled explicitly.
-recommendationMessage=For such reasons, we recommend to: \
+recommendationMessage=For such reasons, we recommend you: \
 <ul>\
   <li>revoke the tokens that were never used and</li>\
   <li>ask users that are currently using their legacy token to generate a token using the new system and revoke their legacy token</li>\
 </ul>
-UserId=User Id
+UserId=User ID
 UserFullName=User full name
 TokenName=Token name
 NumDaysSinceCreation=Days since creation
-NumOfUse=# of use
+NumOfUse=# of uses
 NumDaysSinceLastUse=Days since last use
 HasFreshToken=Fresh token?
 HasFreshToken_tooltip=A fresh token is one that was created using the new system \
   after the most recent use of the user''s legacy token. \n\
   Note: the creation date of legacy tokens which have never been used are reset every time Jenkins is restarted, \
-  which means that the date may be inaccurate.
-HasFreshToken_ok_tooltip=A fresh token exist for that user
-HasFreshToken_warning_tooltip=No fresh token exist for that user
+  which means that the date will be inaccurate.
```

Comment:
I liked *may* better (not all tokens, just specific ones), or perhaps *… means that **their** date…*?

Reply:
_... means the date for **those** tokens will be inaccurate_?

Reply:
Or that 👍